### DISCUSSION TITLE
Classification Numbers

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#434] [Q&A]
How do we deal with classification numbers in RDA? We have a ton of them in MARC. What are they, exactly? Identifiers for manifestations? Why hasn't RDA dealt with classification more directly? @GordonDunsire @AdamSchiff @szapoun @corialanus @SitaKB @briesenberg07 @JianPLee @junghaelee @pennylenger @lake44me

**Comment by AdamSchiff** [comment]
The subject part of an LC classification or Dewey number is "has subject"

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________
From: Crystal Yragui (Clements) ***@***.***>
Sent: Wednesday, November 22, 2023 6:03 PM
To: uwlib-cams/MARC2RDA ***@***.***>
Cc: Adam L Schiff ***@***.***>; Mention ***@***.***>
Subject: [uwlib-cams/MARC2RDA] Classification Numbers (Discussion #434)


How do we deal with classification numbers in RDA? We have a ton of them in MARC. What are they, exactly? Identifiers for manifestations? Why hasn't RDA dealt with classification more directly? @GordonDunsire<https://urldefense.com/v3/__https://github.com/GordonDunsire__;!!K-Hz7m0Vt54!i-Nns7OkeXhuai7Ce4NDR2Yy_Cs3O0YN9BR6HTBtNdiErmvxFXaRDHrOCA8XMA87Zxqs4ydrukJ69LyJFHOmshU$> @AdamSchiff<https://urldefense.com/v3/__https://github.com/AdamSchiff__;!!K-Hz7m0Vt54!i-Nns7OkeXhuai7Ce4NDR2Yy_Cs3O0YN9BR6HTBtNdiErmvxFXaRDHrOCA8XMA87Zxqs4ydrukJ69LyJgAZuRAE$> @szapoun<https://urldefense.com/v3/__https://github.com/szapoun__;!!K-Hz7m0Vt54!i-Nns7OkeXhuai7Ce4NDR2Yy_Cs3O0YN9BR6HTBtNdiErmvxFXaRDHrOCA8XMA87Zxqs4ydrukJ69LyJuVoXjUY$> @corialanus<https://urldefense.com/v3/__https://github.com/corialanus__;!!K-Hz7m0Vt54!i-Nns7OkeXhuai7Ce4NDR2Yy_Cs3O0YN9BR6HTBtNdiErmvxFXaRDHrOCA8XMA87Zxqs4ydrukJ69LyJzqWmI4Y$> @SitaKB<https://urldefense.com/v3/__https://github.com/SitaKB__;!!K-Hz7m0Vt54!i-Nns7OkeXhuai7Ce4NDR2Yy_Cs3O0YN9BR6HTBtNdiErmvxFXaRDHrOCA8XMA87Zxqs4ydrukJ69LyJ94jAOgQ$> @briesenberg07<https://urldefense.com/v3/__https://github.com/briesenberg07__;!!K-Hz7m0Vt54!i-Nns7OkeXhuai7Ce4NDR2Yy_Cs3O0YN9BR6HTBtNdiErmvxFXaRDHrOCA8XMA87Zxqs4ydrukJ69LyJNR97QCM$> @JianPLee<https://urldefense.com/v3/__https://github.com/JianPLee__;!!K-Hz7m0Vt54!i-Nns7OkeXhuai7Ce4NDR2Yy_Cs3O0YN9BR6HTBtNdiErmvxFXaRDHrOCA8XMA87Zxqs4ydrukJ69LyJi15NzKk$> @junghaelee<https://urldefense.com/v3/__https://github.com/junghaelee__;!!K-Hz7m0Vt54!i-Nns7OkeXhuai7Ce4NDR2Yy_Cs3O0YN9BR6HTBtNdiErmvxFXaRDHrOCA8XMA87Zxqs4ydrukJ69LyJNS6w330$> @pennylenger<https://urldefense.com/v3/__https://github.com/pennylenger__;!!K-Hz7m0Vt54!i-Nns7OkeXhuai7Ce4NDR2Yy_Cs3O0YN9BR6HTBtNdiErmvxFXaRDHrOCA8XMA87Zxqs4ydrukJ69LyJnaE2-Z4$> @lake44me<https://urldefense.com/v3/__https://github.com/lake44me__;!!K-Hz7m0Vt54!i-Nns7OkeXhuai7Ce4NDR2Yy_Cs3O0YN9BR6HTBtNdiErmvxFXaRDHrOCA8XMA87Zxqs4ydrukJ69LyJZ66SQ9g$>

**Comment by GordonDunsire** [comment]
Q: Why hasn't RDA dealt with classification more directly?

A: Subject description and access has been treated as a separate discipline from resource description and access for the last hundred years. This is reflected in the separate standards for subject access points and identifiers (aka classification numbers), and their rudimentary treatment in the main cataloguing 'codes', including AACR/2 and the original RDA. This is not just the Anglo-American cataloguing tradition; IFLA has separate sections for 'Cataloguing' and 'Subject Analysis and Access', and promotes ISBD for resource description and Universal Decimal Classification (UDC) for subject access.

Why? In the development of LRM, FRSAD (FR for Subject Authority Data) was generalized as the Nomen/appellation model. A subject heading or classification number is treated the same as an access point (heading) or identifier: as a nomen string assigned to an instance of an entity (tangible or conceptual). The entity class must be a sub-class of Res by definition, as an 'entity in the universe of discourse'. The LRM does not offer any entity classes purely for subjects. RDA follows suit (and AACR2), but provides a refinement of LRM-R12 "has as subject" (domain Work; range Res):

- rdaw:P10256 "has subject" // no range, because RDA does not implement Res
- > rdaw:P10324 "has subject RDA entity" // range RDA Entity, a less-than-Res super-class that enables Res functionality (inheritance, etc.)
- >> rdaw:P10319 "has subject agent" // range Agent
- >> ditto for all RDA entity classes

RDA also refines the appellation element into name/title, access point, and identifier types; these accommodate uncontrolled subject keywords, subject headings, and classification numbers respectively.

What else is there to say about a subject topic, other than appellation, that is helpful to the functions of the catalogue (aka user tasks) as an information retrieval service? A description of the topic (in addition to a human-readable appellation) tips the catalogue into dictionary, encyclopedia, and taxonomy territory, and that's not the product that cataloguers are paid to create (however tempting the rabbit-hole may be ;-). Another factor is ontology design: LRM and RDA avoid unnecessary class proliferation, and prefer to categorize and refine classes with properties and value vocabularies (simple knowledge organization systems). Adding an entity class to RDA is expensive: addition of entity pair-wise high-level relationships; re-organization of Toolkit and Registry menus and displays; documentation for additional complexity in workflows, policy statements, etc. It has to be justified beyond subject authority control, which is accommodated by Nomen.

So, RDA deals directly with subject **headings**, in general and for RDA entities, but it does not deal with subjects (taxonomies).

**Comment by CECSpecialistI** [comment]
[Work] has subject "classification number"

source = data provenance

how to record?

**Reply by CECSpecialistI** [reply]
Data provenance would reify statements? 

Another option: mint nomens and give sources. This would proliferate nomens. How do we feel about that? We decided to do it for other identifiers.

Another option: add boilerplate to strings, such as "LCC: ". This could make parsing difficult later?

I'd like to be able to assign these to @pennylenger next week. @lake44me @GordonDunsire

**Comment by lake44me** [comment]
You're going to have to mint a URI for the triple to reify it, right?  Unless you propose using RDF Star https://www.ontotext.com/knowledgehub/fundamentals/what-is-rdf-star/ - which isn't the standard - yet.   So we wouldn't be "saving triples" and in terms of bytes probably the triple statements would take up more space than just minting a nomen.

**Reply by lake44me** [reply]
Also is a classification number an identifier? Well, sort of for a subject area, although with the "cutter" LC Classifications are supposed to be unique to a manifestation/expression.

**Reply by CECSpecialistI** [reply]
I share Laura's question about the number of triples needed to reify metadata statements here vs. just minting nomens... @GordonDunsire , when you say this is data provenance, what do you envision that data provenance looking like?

**Reply by GordonDunsire** [reply]
For classification numbers taken from a value vocabulary that covers 'all topics', such as DDC, UDC, and LCC, I think the best way of recording provenance is to subtype the "has subject" relationship element: 'has DDC topic', etc. The definition would include the provenance: 'A subject topic from the Dewey Decimal Classification', etc. RDA won't do this on neutrality and scope grounds. The second best way is to prefix the number with a code for the scheme: 'DDC: 021'. This may or may not conform to the scheme's usage guidelines. The third best option is to reify the classification number as a nomen.

A classification number is an identifier for a concept that is used as a the subject topic of a work. The concept is the 'thing', with its caption/subject heading as a 'preferred label' and its classification number as a 'notation' (this is SKOS terminology). A skos notation is equivalent to a concept identifier.

Many of these classification schemes are intended to be synthetic: basic numbers are used as the building blocks of more specific numbers. DDC is an example: a classification number is synthesized from a basic (hierarchical) concept that is qualified with standard subdivisions. The base concepts and the subdivisions are separately notated/enumerated, but the resulting synthetic topic is not. Since nearly every basic concept/topic can be qualified by more than one subdivision, there are more non-notated concepts than notated concepts. This presents a scaling problem for minting IRIs for classification numbers as nomen strings. It would be wonderful to develop and manage an operational system for doing so, because then we could de-reference the number to get the subject heading. but I think it will be expensive and is out of scope for the transform.
If UW is able to support the M21 value vocabularies, then it should be able to support a small vocabulary of classification schemes as subtypes of 'has subject'. Note, however, that differential treatment is required for classification numbers of persons, places, and other RDA entities.

**Comment by CECSpecialistI** [comment]
I propose that we make these identifiers for manifestations, mint nomens for them, and give them "scheme of nomen" triples to show which classification schemes they come from. Figuring out what is a subject portion and what is a manifestation portion is really difficult and isn't just based on what side of a $b an identifier is on. Eh? Am I missing something obvious that would allow us to treat these differently short of minting new properties for each classification system as Gordon suggests above?

**Reply by GordonDunsire** [reply]
@CECSpecialistI: a classification pertains to the content of a manifestation, but not the manifestation itself (different manifestation can carry the same content). So a classification number cannot be an identifier of manifestation except when the classification scheme is designed as an identifier scheme for manifestation and supplies a different number for each edition, etc. In practice, this exception never occurs.

Similarly, the only time there is a "manifestation part" of a classification is a) when the embodied work is about another manifestation, such as the First Folio and b) the other manifestation has a classification number in the scheme. In practice, this is very rare. I'm not sure what you mean otherwise by "manifestation portion".***

A classification number must be a) a Work element (abstraction of content) and b) an identifier for something that is the subject of a work. The vast majority of recorded classification numbers identify a concept that is not an RDA/LRM entity such as Manifestation, Place, or Person, because the vast majority of works are about "stuff".

*** The classification-as-identifier is generally the value of subfield $a only. Some tags, such as 080, encode subdivisions, qualifiers, etc. of the main classification in specific subfields such as subfield $x; these should be concatenated with subfield $a to get the full classification. The other subfields pertain to shelfmarking functions or data provenance.

**Comment by AdamSchiff** [comment]
I think what Crystal meant by "manifestation portion" was the shelflisting portion of a call number that serves to put works in alphabetical order on the shelf by title or by author and title (i.e. the Cutter number in LCC).  That part does not reflect the work's subject.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Comment by tmqdeborah** [comment]
But then, isn’t the shelflisting portion of a call number about the order of an item on a particular library’s shelf?

Deborah

-  -  -  -  -  -  -  -
Deborah Fritz
***@***.******@***.***>

**Reply by CECSpecialistI** [reply]
Thanks for clarifying, Adam. That is what I meant by "manifestation portion".

**Reply by CECSpecialistI** [reply]
Yes, it's the portion that determines the order of the item on a particular library's shelf that I'm referring to. These numbers are really important for libraries to share in copy cataloging/quick cataloging workflows, but aren't about the subject of the work. Hmm. Maybe we need to discuss this during a meeting to get a clear path forward. A good percentage of the tags we have left to map are classification tags.

**Comment by AdamSchiff** [comment]
True.  With shared cataloging, many libraries use the same Cutter but it can be adjusted to fit a particular library, so it does have item attributes.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Comment by CECSpecialistI** [comment]
Maybe we should go with Gordon's suggestion to mint new properties for these. UW already started to do this during LD4P2 with LCC. We could expand that vocabulary.

**Comment by AdamSchiff** [comment]
“In practice, this exception never occurs.“

Gov document classifications are possibly an example.  They identify the government agency and the title of the document.  The same content issued by a different agency of the government would get a different classification number.

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
Box 352900
Seattle, WA 98195-2900
aschiff @ uw.edu
________________________________

**Reply by GordonDunsire** [reply]
My bad use of phrasing. I should have said that "classification" and "identification" are treated separately in LRM/RDA. I think this follows other IFLA developments, such as the change from "Classification and Indexing Section" to "Subject Analysis and Description section" in 2016. Classification is subject analysis; identification is the assignment of an identifier that distinguishes one instance of an entity from another. Both are treated separately from "categorization", which is the analysis of sub-groupings of instances based on overall common values of characteristics within a single entity.

So I guess, with legacy data, "classification" values should be separated into subject, identifier, and category parts for some schemes. The category/genre collision is particularly interesting.

But maybe we can get away with shifting the burden of semantic coherency to the schemes; that is, subtyping the subject property for each scheme and using the complete "classification" number as the value. Otherwise, we may not be able to get back out of the rabbit-hole.

**Comment by GordonDunsire** [comment]
Here's my initial analysis of the MARC 21 classification number fields.

[Transforming subject classification number data from MARC 21 to RDA.docx](https://github.com/uwlib-cams/MARC2RDA/files/14194400/Transforming.subject.classification.number.data.from.MARC.21.to.RDA.docx)

Option 3 uses SKOS directly, rather than RDA Nomen (because Nomen is defined in relation to other RDA entities, and not "all things" (i.e. LRM Res).

@tmqdeborah calculated some totals of occurrences of codes for classification schemes from the LoC list. Bliss Classification and BISAC Subject Headings (a misnomer?) are the next highest after DDC, LCC, and UDC. There is evidence of import or retroconversion errors, such as "lacc" when "lcc" is presumably intended, and the use of codes that do not appear in the LoC list. More analysis is required to support option 2, especially on MARC 21 databases from outside of the USA/LoC.

I think the cleanest/best option is 3, but this will required the generation and maintenance of millions of instance URIs as a Concepts dataset.

**Reply by GordonDunsire** [reply]
Oops, just found the id.loc.gov URIs for the classification scheme list; this can be used to support option 3.

**Comment by CECSpecialistI** [comment]
I favor option 3.

**Reply by GordonDunsire** [reply]
I have updated the first draft and added some detail about option 3. It's now in the shared Google drive at:

https://docs.google.com/document/d/1T5VyAH6bPKBTJp_j4l2ecOLUlP00u-GK7Ubti58mtqU/edit?usp=sharing

Please add comments, etc. to the doc.

**Reply by GordonDunsire** [reply]
I think we should expand the scope of this discussion to the 6XX fields. RDA makes little distinction between subjects as classifications, codes, thesauri, or controlled headings. Similar issues apply to usage of codes from [Subject Headings and Term Source Codes](https://www.loc.gov/standards/sourcelist/subject.html) and their RDF representation in [Subject Schemes](http://id.loc.gov/vocabulary/subjectSchemes). See the Google doc for ongoing discussion.

**Reply by CECSpecialistI** [reply]
This makes sense to me. I've added this as our main agenda item for tomorrow's meeting.

**Comment by pennylenger** [comment]
@GordonDunsire Hi, Gordon. I carefully read the document you wrote and found it very informative. However, I have some questions which might seem naitive. I would appreciate your answers if you have the time. I I've copied your document and commented mainly in 3A part to avoid messing up the original document.
https://docs.google.com/document/d/15LLdixSsj7u91tRhZDKuo2piu5HR23D4-Lzbq6dDa3o/edit?usp=sharing

**Reply by pennylenger** [reply]
@GordonDunsire Thank you very much for providing such a detailed and thorough answer to my questions. It's very helpful. I really appreciate your time and guidance.

**Comment by pennylenger** [comment]
@GordonDunsire Hi, Gordon, I have another question for 050, do the first and second indicator matter? should they be mapped differently?

First Indicator:
Existence in LC collection
#- No information provided
0 - Item is in LC
1 - Item is not in LC

Second Indicator:
Source of call number
0 - Assigned by LC
4 - Assigned by agency other than LC

And for the first indicator #, the explanation is kind of strange: 
#- No information provided:
Used for all call numbers assigned by agencies other than the Library of Congress.

For the # in first indicator and 4 in the second indicator Assigned by agency other than LC, should the mapping be modified to other agency?   
Thank you for your help.

**Reply by GordonDunsire** [reply]
@pennylenger: The first indicator of field 050 is irrelevant because it is essentially a statement about an item that exemplifies the manifestation being described. We decided not to mint any items from the classification and subject fields because there is insufficient information to identify them.

The second indicator is a statement of assignment, but 'Assignment provenance is excluded from this analysis, and is part of the larger issue of statement reification' as the doc says.

For the classification fields, our basic model creates an instance of skos:Concept which then links to statements about the concept. 'Assignment' of the concept to a work requires a reification of the statement 'This work-has classification-that concept' so that it can be linked to the assigning agent.

To respond to your comment that first indicator # is strange: this is an encoding for MARC 21 records created outside of LC and imported into union catalogues. These records do not necessarily indicate, in their overall values, if the assigning agency holds an exemplar, etc. of the work in its collection: 'no information provided'.

In short, the indicators of field 050 do not matter for this part of the transform.
